freqtrade/optimize/hyperopt.py

```diff
@@ -339,6 +354,9 @@ def start(self) -> None:
         logger.info(f'Number of parallel jobs set as: {config_jobs}')
 
         opt = self.get_optimizer(config_jobs)
+
+        colorama_init()
```

Comment:
according to the colorama docs, when calling `init()` - we should call `deinit()` too, to reset the terminal (stdout, stderr) to how it was.

Reply:
`deinit()` is needed "to stop using colorama before your program exits" (cited from the docs). We do not need to stop using colorama *before* freqtrade exits. we are stopping it *at* the program exit (automatically).